lib/types/db.ts

```diff
@@ -41,3 +44,16 @@ export type PairAttributes = PairFactory & {
 };
 
 export type PairInstance = PairAttributes & Sequelize.Instance<PairAttributes>;
+
+export type ReputationEventFactory = {
+  event: ReputationEvent;
+  nodeId: number;
+};
+
+export type ReputationEventAttributes = ReputationEventFactory & {
+  id: number;
+};
+
+export type ReputationEventInstance = ReputationEventAttributes & Sequelize.Instance<ReputationEventAttributes> & {
+  createdAt: number;
```

Comment:
Do we need this here? Or is it fine to just leave `createdAt` in the database, retrievable by query if we need it. I believe that's similar to our approach with `Nodes`.

Reply:
If we want to prune old events on startup I would leave it as is in order to get all needed information in one query.